cmd/debugger/main.go

```diff
@@ -66,12 +66,18 @@ func handleWinChangeData(ptmx *os.File, data []byte) error {
 }
 
 func populateShellHistory(cmd string) error {
-	f, err := os.Create("/root/.bash_history")
-	if err != nil {
-		return err
+	for _, f := range []string{
+		"/root/.ash_history",
```

Comment:
Do we need to also `MkdirAll("/root")`?

Reply:
I don't think so, if that were the case the image would be very broken I suspect.